packages/block-library/src/post-excerpt/index.php

```diff
@@ -18,15 +18,16 @@ function render_block_core_post_excerpt( $attributes, $content, $block ) {
 		return '';
 	}
 
-	$more_text = isset( $attributes['moreText'] ) ? '<a class="wp-block-post-excerpt__more-link" href="' . esc_url( get_the_permalink( $block->context['postId'] ) ) . '">' . $attributes['moreText'] . '</a>' : '';
+	$more_text           = ! empty( $attributes['moreText'] ) ? '<a class="wp-block-post-excerpt__more-link" href="' . esc_url( get_the_permalink( $block->context['postId'] ) ) . '">' . $attributes['moreText'] . '</a>' : '';
+	$filter_excerpt_more = function( $more ) use ( $more_text ) {
+		return empty( $more_text ) ? $more : ' [&hellip;]';
```

Comment:
Having a bit of trouble to understand the logic here, why are we using `[&hellip;]` here?

Reply:
That's just the default value for this filter in core. We could remove this default `hellip` altogether.

Reply:
ok, so basically what you're doing this is:

If the block has a "more text" attribute, reset the filter. I think it makes sense but why not just return `$more_text` there?

Reply:
Also maybe add some comments to explain the logic of why we're using filters here.

Reply:
>I think it makes sense but why not just return $more_text there?

We wrap this block in a `paragraph` and  have `showMoreOnNewLine` that also uses `paragraph`, so we can't have these two nested. We don't always want `more text` just appended to the excerpt.